build/Targets/Imports.targets

```diff
@@ -140,8 +140,9 @@
             <PublicKey>$(RoslynInternalKey)</PublicKey>
             <PublicKeyToken>fc793a00266884fb</PublicKeyToken>
           </PropertyGroup>
-          <!-- Real-signing is only supported cross-platform in the boostrap toolset -->
-          <PropertyGroup Condition="'$(OS)' != 'Windows_NT' AND '$(BootstrapBuildPath)' == ''">
+          <!-- Real-signing cross-platform currently has a blocking bug:
+               https://github.com/dotnet/roslyn/issues/23521 -->
```

Comment:
The deep irony of disabling signing while simultaneously fixing the bug.... :smile: